.github/CODEOWNERS

```diff
@@ -4,3 +4,4 @@
 /media/starboard/  # no owners
 /starboard/  # no owners
 /components/viz/service/display/starboard/  # no owners
+/cobalt/build @andrewsavage1 @niranjanyardi @kaidokert
```

Comment:
Do you just want to set up a "Build" team or such ? It's easy to create one, i'd rather prefer not to have user IDs

Reply:
I made you one here: https://github.com/orgs/youtube/teams/cobalt-build/members

You, Andrew, Oscar for now - feel free to add / remove people.

Reply:
I suggested using user ids because I thought it would just be for the gn.py file, does the rest of the directory need owners? If it is for one file, I don't see the downside of using user ids, why do you want to avoid it @kaidokert ?

Reply:
Can do this in a seperate PR if needed. Closing conversation blocking the PR